README.md

```diff
@@ -130,13 +130,31 @@ This repository contains snapcraft packaging for Pelion Edge on Ubuntu.
     sudo snap connect pelion-edge:network-observe :network-observe
     sudo snap connect pelion-edge:process-control :process-control
     sudo snap connect pelion-edge:shutdown :shutdown
+    sudo snap connect pelion-edge:dbus-wpa wpa-supplicant:service
```

Comment:
Should add this connect option to the connect.sh script

Reply:
Got it. Will make the required change.